idd/Energy+.idd.in

```diff
@@ -448,19 +448,35 @@ PerformancePrecisionTradeoffs,
       \memo This object enables users to choose certain options that speed up EnergyPlus simulation,
       \memo but may lead to small decreases in accuracy of results.
   A1, \field Use Coil Direct Solutions
-      \note If Yes, an analytical or empirical solution will be used to replace iterations in
+      \note If Yes, an analytical or empirical solution will be used to replace iterations in 
       \note the coil performance calculations.
       \type choice
       \key Yes
       \key No
-      \default No
```

Comment:
@JasonGlazer Was this default intentionally deleted? If so, why?

Reply:
I don't know why it would have been deleted.

Reply:
Look at the comment on the line just below.  Originally the new field modifications were pasted into the wrong block and left a second `\default` on the wrong field.  This duplicate was remedied, but it looks like this default was lost in the process.

Reply:
I've got a couple more test runs to make sure this works at least well-enough to go in, then I'll fix the default and push.